src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/controller/isl/IslFsm.java

```diff
@@ -165,6 +169,10 @@ public void updateMonitorsAction(IslFsmState from, IslFsmState to, IslFsmEvent e
             fireBecomeStateEvent(context);
         } else if (isSyncRequired) {
             fire(IslFsmEvent._FLUSH);
+
+            if (discoveryBfdMonitor.bfdIsEnabled()) {
```

Comment:
bfd monitor ... bfd is enabled... too many "bfd", as for me. `isEnabled()` ... no `isOperational()` should be enoug.

Reply:
Fixed.